SPECS/libreswan/libreswan.signatures.json

```diff
@@ -1,8 +1,5 @@
 {
  "Signatures": {
-  "libreswan-4.7.tar.gz": "ddd6337b3900063d870301c3d9f61f56107c765850fb00a163d360359ff3fc44",
-  "ikev1_dsa.fax.bz2": "030c7ac59422c2c36ed332efca925b18dd842da138619daab65c15663c687086",
-  "ikev1_psk.fax.bz2": "4698f0d8e653e20f279dd54aa1a270aadcc3e99da9e3a91852af12a644cc3531",
-  "ikev2.fax.bz2": "36f356538cdcab6a1712425ad386f32834f0dc333acbc4cc642db0f910be1d21"
```

Comment:
This looks like a bug inside `libreswan.spec` where we conditionally exclude three sources for non-test runs. This makes us generate different source RPMs for the same spec depending on set macros, which is not what we want. Especially, since published RPMs are built with tests (and network access) disabled, so we're going to end up publishing SRPMs with three missing sources and then we're going to fail building the package from these SRPMs, if we want to run the build with tests.

In other words: let's update the spec by removing `%if %{with_check}` from it and keep the signatures.

Reply:
Oh, I think this was caught in the crossfire while I was updating some other signatures, but I can just knock that out while here.